CMSgov/mint-app
# [EASI-4301] Make the Navigation Sticky

# EASI-4301


## Changes and Description

- Deleted unused `HeaderWrapper` component --- to remove bloat
- Adjusted `index.scss` to better match what is needed 
- Updated test to better reflect the actual component
- Shifted the header components around to make the sticky nav to work 
    - Added comments to help future development work
- `src/components/NavigationBar/index.scss` was mainly prettier formatting update
- Updated props for `NavigationBar` component
- Added `z-top` style class for the `SkipNav` component



## How to test this change

1. Check the functionality and stickiness works for both desktop size and mobile size for the following pages:
    1. Landing Page
    2. Get Access page
    3. Sign in Page
    4. NDA page
    5. rest of the Logged in pages
   
  





## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
